zap/src/main/java/org/zaproxy/zap/extension/stdmenus/PopupContextTreeMenu.java

```diff
@@ -90,6 +95,16 @@ public boolean isEnabledForContext(int contextId) {
         return true;
     }
 
+    /**
+     * Returns whether or not the menu should be enabled when multiple contexts are currently
+     * selected, override if not
+     *
+     * @return
```

Comment:
Please complete this, add a period to the first sentence, and mention that the default is true. Also it would be great if you added a since tag `@since TODO Add version` (we'll do a search and replace before the next release).

Reply:
added!